doc/classes/Basis.xml

```diff
@@ -131,6 +131,7 @@
 				- The [member Vector3.y] contains the angle around the [member y] axis (yaw);
 				- The [member Vector3.z] contains the angle around the [member z] axis (roll).
 				The order of each consecutive rotation can be changed with [param order] (see [enum EulerOrder] constants). By default, the YXZ convention is used ([constant EULER_ORDER_YXZ]): Z (roll) is calculated first, then X (pitch), and lastly Y (yaw). When using the opposite method [method from_euler], this order is reversed.
+				[b]Note:[/b] This method assumes the basis is [i]uniform[/i] (all axes share the same length) (see [method get_scale]).
```

Comment:
This closes https://github.com/godotengine/godot/issues/90727 
However, the alternative is to move this information on top. I don't think it deserves to be a note (without providing a workaround) because you absolutely need to know this information to use this method properly.

> Returns this basis's rotation as a [Vector3] of [url=https://en.wikipedia.org/wiki/Euler_angles]Euler angles[/url], in radians. To prevent unexpected results, the basis must be [i]uniform[/i] (all axes share the same length) (see [method get_scale]).

or, likely too subtle:
> Returns this [i]uniform[/i] basis's rotation as a [Vector3] of [url=https://en.wikipedia.org/wiki/Euler_angles]Euler angles[/url], in radians.

Reply:
Is uniform well-defined somewhere?

We have different definitions of uniform, uniform scale, orthogonal vs is only a rotation.

Reply:
"Uniform" in this context is defined in Basis' leading description among others:

> A [Basis] is [b]orthogonal[/b] if its axes are perpendicular to each other. A basis is [b]normalized[/b] if the length of every axis is [code]1[/code]. A basis is [b]uniform[/b] if all axes share the same length (see [method get_scale]). A basis is [b]orthonormal[/b] if it is both orthogonal and normalized, which allows it to only represent rotations. A basis is [b]conformal[/b] if it is both orthogonal and uniform, which ensures it is not distorted.

Reply:
Side note, but that intro section might be clarified with something like:

```
A [Basis] is:
- [b]orthogonal[/b] if its axes are perpendicular to each other.
- [b]normalized[/b] if the length of every axis is [code]1[/code].
- [b]uniform[/b] if all axes share the same length (see [method get_scale]). 
- [b]orthonormal[/b] if it is both orthogonal and normalized, which allows it to only represent rotations. 
- [b]conformal[/b] if it is both orthogonal and uniform, which ensures it is not distorted.
```

I don't know if a split sentence like that translates well, though.

Reply:
Lists are awesome but currently they take too much space in the built-in documentation because unordered lists are not supported. Each new line adds a whole other line to simulate a paragraph. When I made the PR originally I was probably accounting for this. But... sure

Reply:
Ah, good to know. Unordered lists are used in a few places for parameters in method descriptions, but I think the vertical space is more acceptable there.

Reply:
Yeah, but that's accidental because `- ` is not escaped and it conveniently denotes a list for that markdown format. We should still do "lists" like this for general consistency. Look at https://github.com/godotengine/godot/pull/88900 , too.

Reply:
Maybe worth linking to `[method orthonormalized]`?

Reply:
Sure. I assume you're talking about `get_euler()`'s description? But it's also worth mentioning in the leading description I guess.

Now that I think about it, for `get_euler()` to return correctly, it needs a _conformal_ basis and not just _normalized_, does it not?